### ISSUE TITLE
Env variable TRACING_SYSTEM cannot be blank

### ISSUE THREAD

**Issue by HontoNoRoger** [strimzi/client-examples#85]
We just got a crash looping container for the `quay.io/strimzi-examples/java-kafka-consumer:latest` image, as the newly introduced env variable `TRACING_SYSTEM` cannot be blank.

The error we see is:
```
Exception in thread "main" java.lang.NullPointerException
	at TracingSystem.forValue(TracingSystem.java:12)
	at KafkaConsumerConfig.fromEnv(KafkaConsumerConfig.java:75)
	at KafkaConsumerExample.main(KafkaConsumerExample.java:22)
```

It seems the tracing system needs to be set currently, otherwise the application won't run: https://github.com/strimzi/client-examples/blob/main/java/kafka/consumer/src/main/java/KafkaConsumerConfig.java#L75

**Comment by scholzj** [comment]
@OwenCorrigan76 Can you please look at this?

**Comment by ppatierno** [comment]
Closed via #86
